### PR TITLE
[v7r1] use createClient decorator

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*RSS
FIX: use createClient decorator for RSS clients

ENDRELEASENOTES

### CODE REVIEW

File: Core/Utilities/ThreadSafe.py [hunk 1]
```diff
@@ -36,8 +35,8 @@ def unlock(self):
     return self.__lock.release()
 
 
-# FIXME: not used
-class WORM:
+# FIXME: not used UNUSED
```
Comment (chaen) [inline Core/Utilities/ThreadSafe.py:38]: why not deleting it then ?

File: Core/Utilities/TypedList.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
+"""  UNUSED UNUSED UNUSED UNUSED UNUSED
```
Comment (chaen) [inline Core/Utilities/TypedList.py:1]: remove it ?

File: ResourceStatusSystem/Client/ResourceManagementClient.py [hunk 3]
```diff
@@ -1,12 +1,11 @@
 """ ResourceManagementClient
 
-  Client to interact with the ResourceManagementDB.
-
+  Client to interact with the ResourceManagement service and from it with the DB.
```
Comment (chaen) [inline ResourceStatusSystem/Client/ResourceManagementClient.py:3]: Maybe you should be a bit more specific  in the differences between `ResourceManagement.py` and `ResourceManagementClient.py` ? Even the docs are the same :-D

File: Core/Utilities/ThreadSafe.py (hunk 1 above)
Comment (fstagni) [reply Core/Utilities/ThreadSafe.py:38]: I'll open a PR now for `integration` for removing this stuff.

File: ResourceStatusSystem/Client/ResourceManagementClient.py (hunk 3 above)
Comment (fstagni) [reply ResourceStatusSystem/Client/ResourceManagementClient.py:3]: Done.

File: Core/Utilities/TypedList.py (hunk 2 above)
Comment (fstagni) [reply Core/Utilities/TypedList.py:1]: https://github.com/DIRACGrid/DIRAC/pull/4796

File: ResourceStatusSystem/Client/ResourceManagementClient.py (hunk 3 above)
Comment (chaen) [reply ResourceStatusSystem/Client/ResourceManagementClient.py:3]: OK much clearer. And just for my education: in general, we should use `ResourceClient`, correct ? Only if we specifically want to query RSS should we use the other one ?

Comment (fstagni) [reply ResourceStatusSystem/Client/ResourceManagementClient.py:3]: Yes, `ResourceClient` is in practice the API for RSS, and normally the only object you want to use.

**Review by chaen — APPROVED** [review]

File: Core/Utilities/Statistics.py [hunk 4]
```diff
@@ -1,6 +1,5 @@
-##################################################################################################
```
Comment (atsareg) [inline Core/Utilities/Statistics.py:1]: Delete if not used ? I guess we have numpy in DIRACOS that has this stuff and more

**Review by atsareg — APPROVED** [review]

File: Core/Utilities/Statistics.py (hunk 4 above)
Comment (fstagni) [reply Core/Utilities/Statistics.py:1]: I deleted it in v7r2

**Pull request merged**